src/docgen/transpile/csharp.ts

```diff
@@ -103,7 +103,7 @@ export class CSharpTranspile extends transpile.TranspileBase {
     const type = this.type(struct);
     const indent = ' '.repeat(4);
     const inputs = struct.allProperties.map((p) => {
-      return `${indent}${this.formatFnParam(this.property(p))}`;
+      return `${indent}${this.formatFnParam(this.parameter(p))}`;
```

Comment:
Can you explain? `struct.allProperties` returns `reflect.Property` - so why do we have to call `this.parameter` instead of `this.property`?

Reply:
My mistake here, this is unnecessary - reverted to the original `this.property(p)`